### ISSUE TITLE
Replace Logo

### ISSUE THREAD

**Issue by TheoBr** [TheoBr/Chrometana#29]
In the pile of spear phishing emails I get every day about Chrometana, I managed to miss one that was a sketchy-looking but apparently legitimate complaint from Microsoft about the logo. It's too similar to Cortana's so we need something more unique.

I don't have time to design something right now so if I don't have something by Wednesday I'm just gonna make it a black dot. 

I'd love for something tongue-in-cheek, like a bent up paperclip. I'm not picky though. This is open for anyone to submit their designs, so submit away!

**Comment by bennyty** [comment]
![microsoft-logo1](https://user-images.githubusercontent.com/592150/27989955-ff7e3090-6414-11e7-879c-75a97872e924.jpg)

**Comment by Preston12321** [comment]
![chrometana_transparent](https://user-images.githubusercontent.com/13774118/28184803-6cbb3d42-67da-11e7-8b19-08ea580bad86.png)

**Comment by TheoBr** [comment]
@preston12321 I really like your design but I'm petrified of getting another DMCA. Any chance you could do more to make it look a little less Google-y? Even just straightening the diagonal color cuts would probably be enough 

Thank you so much for your submission! I really hope we can use this

**Comment by Preston12321** [comment]
@TheoBr In hindsight, I agree it might have still been dangerous, so I straightened the cuts, rearranged the colors, and took the liberty of darkening them as well just to be safe. Wasn't too sure about changing the colors, so I made one without the change to compare. Do with these images what you wish.

<img src="https://user-images.githubusercontent.com/13774118/28200103-9be0641c-682f-11e7-8d5f-73a8f8c2489f.png" alt="Original Colors" width="50%">
<img src ="https://user-images.githubusercontent.com/13774118/28200104-9beec5ca-682f-11e7-85fe-8970595fd2ff.png" alt="Darkened Colors" width="50%">
